models/mixpanel/model.sql

```diff
@@ -0,0 +1,92 @@
+create or replace view {{env.schema}}.mixpanel_engage as
+(
+	select
+		mp_reserved_created			as ac_user_created,
```

Comment:
why are we using the prefix `ac_` here? we're not using that elsewhere. seems like this should just be `user_created`, and all the others likewise. there `$` convention that we had been using isn't relevant here because this isn't a cross-domain analysis, it's just for mixpanel.